### ISSUE TITLE
Skewed data handling

### ISSUE THREAD

**Issue by didkivskyy** [BrainJS/brain.js#694]
What would be a recommended approach to training on highly skewed data?

I'm trying to train a **feed forward NN** mainly for classification purposes. I have about 20 input values. My output contains two properties (classes) both of which are equal to _zero_ 99% of the time. The data is well normalized.
As one would expect with most kinds of regressions I get low _training error_ and high _accuracy_ but awful _recall_ level.
Basically that 1% of outpul values being equal to _one_ is almost never predicted by the network.

What would be the best way to fight this?

I've seen some articles about using **samples weights** for such cases when using **keras**. But i'm not sure if it's somehow supported by Brain.js

Any ideas are appreciated.
Thanks.

**Comment by robertleeplummerjr** [comment]
I'd simplify your training data, and remove very similar data within a certain threshold.  For example, if 10000 training values vary by 0.01, use a subset of them.  Perhaps 100 or so.  Something else you could try is adjust the `hiddenLayers` and `learningRate`.
